app/src/main/res/layout/error_panel.xml

```diff
@@ -72,4 +72,19 @@
         android:visibility="gone"
         tools:visibility="visible" />
 
+    <Button
+        android:id="@+id/error_openinbrowser"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="4dp"
+        android:layout_marginBottom="8dp"
+        android:text="@string/error_openinbrowser"
```

Comment:
There is already a string for `Open in browser`: https://github.com/TeamNewPipe/NewPipe/blob/2a98cca80100d87562f0c9ff1fadeac3838f6ff6/app/src/main/res/values/strings.xml#L10

```suggestion
        android:text="@string/open_in_browser"
```